frontend/apps/app/lib/chat/workflow/nodes/answerGenerationNode.ts

```diff
@@ -12,6 +15,184 @@ interface PreparedAnswerGeneration {
   formattedChatHistory: string
 }
 
+// Define schema for JSON Patch operations (RFC 6902)
+const jsonPatchOperationSchema = v.object({
+  op: v.picklist(['add', 'remove', 'replace', 'move', 'copy', 'test']),
+  path: v.string(),
+  value: v.optional(v.unknown()),
+  from: v.optional(v.string()),
+})
```

Comment:
This definition may be used for prompting Build agents to improve the accuracy of their responses. I would like to make this my next pull request.

Reply:
Whether `value` or `from` is required depends on the value of `op`. This is defined in `frontend/apps/app/libs/schema/operationsSchema.ts`. Please take a look in the next PR!

Reply:
I was not aware of it. Thank you!!